backend/infrahub/workflows/constants.py

```diff
@@ -3,6 +3,7 @@
 
 class WorkflowType(InfrahubStringEnum):
     INTERNAL = "internal"
+    CORE = "core"
     USER = "user"
```

Comment:
It is possible to add a comment to specify meaning difference between different flows, especially `CORE` and `USER`?